.github/workflows/add_to_GA.yml

```diff
@@ -0,0 +1,25 @@
+name: Add issues to GA Project
+on:
+  workflow_run:
+    workflows: ["Label issues"]
+    types:
+      - completed
+  issues:
+    types:
+      - labeled
```

Comment:
What does this new `workflow_run` mean?
Don't we run this workflow when a new issue is created or an existing issue reopened?

Reply:
This solves the concurrency issue of "which workflow will run first"
I also kept the files separate for this implementation as there was no way I found where I could ignore the "if an issue is labelled" case in the "assign triage label" workflow

Reply:
This implies (as far as my understanding goes) that this specific workflow would run only after the triage label is assigned/whenever a issue is labelled-note that there is also a label filter later in the steps
P.S. this workflow will now be changed to handle the new way we triage tickets, and is outdated